src/cephadm/cephadmlib/daemons/nfs.py

```diff
@@ -29,7 +29,7 @@ class NFSGanesha(ContainerDaemonForm):
 
     daemon_type = 'nfs'
     entrypoint = '/usr/bin/ganesha.nfsd'
-    daemon_args = ['-F', '-L', 'STDERR']
+    daemon_args = ['-F', '-L', '/var/log/ganesha.log']
```

Comment:
there's an additional unit test `test_deploy_nfs_containe` that's checking for
```
assert runfile_lines[-1].endswith('quay.io/ceph/ceph:latest -F -L STDERR')
```
that would have to be adjusted with this change, similarly to how you adjusted the other one.

Reply:
Thanks for the catch, making the required change.